TbsCore/Helpers/Classificator.cs

```diff
@@ -201,10 +201,10 @@ public enum HeroItemEnum
             Stackable_Ointment_0, // =106
 
             Stackable_Scroll_0,
-            NonStackable_Bucket_0,
-            NonStackable_Tablets_0,
-            NonStackable_Book_0, // =110
-            NonStackable_Artwork_0,
+            Stackable_Bucket_0,

```

Comment:
These aren't stackable though?

Reply:
@Erol444 hm wait ttwars didnt update this, travian update they are stackable now